spec/support/request_specs.rb

```diff
@@ -1,11 +1,26 @@
+require 'rack/test'
+require 'json_spec'
```

Comment:
Just as a question: is it necessary to explicitly require json_spec again? It should already be loaded from the test group in the Gemfile